ui/src/app/services/app-config.service.spec.ts

```diff
@@ -1,35 +1,27 @@
-
 import { AppConfigService } from './app-config.service';
-import { HttpClientModule } from '@angular/common/http';
-
-import { TestBed } from '@angular/core/testing';
-import { HttpClient } from 'selenium-webdriver/http';
+import { HttpClient } from '@angular/common/http';
```

Comment:
Here's where VSCode (or whatever IDE you use) let you down.

Ok, let's be honest.  Selenium let you down.  😄